### ISSUE TITLE
Program saved with symlink refuses to install in Win7

### ISSUE THREAD

**Issue by gitut2007** [gyng/save-in#111]
Hi.
I downloaded from https://www.foobar2000.org/download last version 1.4.4 , instead of saving it in C\user\downloads, I used a symlink to save it in other Drive\folder.
It downloas fine, then in the target non-C folder I click the exe > the authorization dialog comes 'do you want to allow'... [and shows as if the file were from C] so I accept but I am informed +-'not for your Windows version'....

So I download the same file to the same non-C folder using regular FFox 'Save in' [so it asks to rewrite the file] and now the exe can be installed.

I guess this is a failure of symlinks and nothing your addon can do about? [like to take out the 'C' identifier so the file appears 'normal' as from it's true non-C folder?] 
So, for programs one should always use regular FF way and not use the addon+symlinks?

Thanks.

**Comment by gyng** [comment]
I suspect the file downloaded might be different from the file downloaded normally. Can you see if the filesizes are the same?

**Comment by gitut2007** [comment]
Both are the same except if saved through symlink some sort of 'C\user\\downloads' identifier is injected in the file [I know because in the Win7 pop-up when trying to install the exe it says the file is from that place instead of the true place where the file was saved],  when trying to install the exe this seem to cause Win7 to reject it.
It seems you are on Linux?, otherwise you could download and check the files behaviour in Win7 (the program is very small, 4 Mb) [and is one of the programs that people that go Linux miss]
Thx

**Comment by gyng** [comment]
@gitut2007 Hi, I've tried this on Windows 10, and it seems to work. Is the download being caught by antivirus, or blocked by Firefox/Chrome?

**Comment by gitut2007** [comment]
No antivirus problem, not blocked, download perfect. Anyway the file should not be marked as being from a different folder as where it really is. 
Hopefully someone with win7 would report...

**Comment by gitut2007** [comment]
Saving last Firefox with symlink worked [another issue: it saved a long string name without estension, I had to manually change the file name to 'ff67.exe' ] right.
Hopefully the case with foobar was a one-off, if I have another fail occurrence with another program, I will report.

**Comment by gitut2007** [comment]
Saving last Firefox with symlink worked [another issue: it saved a long string name without extension, I had to manually change the file name to 'ff67.exe' ] right.
Hopefully the case with foobar was a one-off, if I have another fail occurrence with another program, I will report.
